modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -386,6 +388,16 @@ func (t TLSConfig) MakeTLSClientConfig(ctx caddy.Context) (*tls.Config, error) {
 		cfg.RootCAs = rootPool
 	}
 
+	// Renegotiation
+	switch t.Renegotiation {
+	case "never":
+		cfg.Renegotiation = tls.RenegotiateNever
```

Comment:
Since this is the default, is it needed? Maybe we should only accept "once" or "freely". Make sure to reject invalid values too.

Reply:
"never" is _currently_ the default, and perhaps forever. But I think that it's better to let users setting explicitly to "never" to protect them from any default value change. But I'll you decide.
Invalid values are rejected now.